demo/admin/src/products/ManufacturersGrid.tsx

```diff
@@ -64,16 +64,15 @@ export function ManufacturersGrid() {
             width: 150,
             renderHeader: () => (
                 <div style={{ display: "flex", alignItems: "center" }}>
```

Comment:
@jamesricky shouldn't this be done in the theme? Or is this even necessary?

Reply:
> Or is this even necessary?

Good point, this shouldn't be necessary, as the parent element of the header (`.MuiDataGrid-columnHeaderTitleContainerContent`) already has exactly those styles by default. 

@andrearutrecht a fragment should be enough here, instead of the styled div.

Reply:
This should probably be removed from the handmade example as well.

Reply:
@andrearutrecht please remove it here as well.